template/client/store/state.js

```diff
@@ -0,0 +1,9 @@
+// @flow
+
+import type { StateType } from '~/types'
+
+export const state: StateType = {
```

Comment:
Exporting `state` is a specific case.
`vuex` docs suggest that we export factory function instead:

```js
function createState (): StateType { ... }

export default createState
```